frontend/utilities/yaml/index.ts

```diff
@@ -31,7 +31,7 @@ export const agentOptionsToYaml = (agentOpts: any) => {
 
   let yamlString = yaml.dump(agentOpts);
   if (addFlagsComment) {
-    yamlString += "command_line_flags: {} # requires Orbit\n";
+    yamlString += "command_line_flags: {} # requires Fleet's osquery installer\n";
```

Comment:
Am curious about the `{}` syntax in YAML, what is it?

Reply:
Hmm, I'm not sure...

I think the goal of `{}` is to signal that `command_line_flags` accepts a dictionary.

It looks like YAML does accept the `{}` format. Check out the 4th code sample in the Ansible docs here:
https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html

Reply:
Gotcha, both following ways are valid:
![Screen Shot 2022-10-13 at 11 24 26](https://user-images.githubusercontent.com/2073526/195623972-ec98674b-1ead-4d5b-8ce1-d9a095986bce.png)
![Screen Shot 2022-10-13 at 11 25 09](https://user-images.githubusercontent.com/2073526/195623978-734249a4-1aaf-4a28-8d49-401d589df2b6.png)